pressbooks/pressbooks-book-directory-fe
# Image fallback

This PR Solves #180 

Solution uses `https://www.npmjs.com/package/v-img-fallback`

# How to test

1. Change  some images urls in your algolia index to see those images being replaced by the fallback cover image
